test/PayID/pay-id-integration-test.ts

```diff
@@ -109,7 +109,7 @@ describe('PayID Integration Tests', function (): void {
 
   it('receipt', async function (): Promise<void> {
     // GIVEN a Pay ID.
-    const payID = '$travel.payid.xpring.money/dino'
+    const payID = 'dino$travel.payid.xpring.money'
```

Comment:
@austin-king will this break after your PR on the refactor is merged? Would these have to be changed to `bob` etc. ?

Reply:
No, it just won't work in general as we never had this value seeded in the DB. We can add one if need be.